specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2018-01-22/examples/DPSGetKey.json

```diff
@@ -0,0 +1,19 @@
+{
+    "parameters": {
+        "provisioningServiceName": "myFirstProvisioningService",
+        "resourceGroupName": "myResourceGroup",
+        "api-version":"2018-01-22",
+        "subscriptionId": "91d12660-3dec-467a-be2a-213b5544ddc0",
```

Comment:
I'd recommend removing real subscription ID from the examples before these reach `master`.

Reply:
I see other teams are including subs in their examples, what is the practice here?

Reply:
Ah, I've seen it happen both ways. I recommend avoiding checking in identifying data, but I may be paranoid. :)

Reply:
These are also private environment subs they do not have any meaning in public azure, since our private env randomly generates a guid